go.mod

```diff
@@ -37,6 +37,7 @@ require (
 require (
 	github.com/FactomProject/basen v0.0.0-20150613233007-fe3947df716e // indirect
 	github.com/FactomProject/btcutilecc v0.0.0-20130527213604-d3a63a5752ec // indirect
+	github.com/Fantom-foundation/go-opera v1.1.3-rc.5 // indirect
```

Comment:
go-opera module should be replaced by sonic submodule.

Reply:
replaced